flow-nodes/api-builder-plugin-typeof/src/flow-nodes.yml

```diff
@@ -5,28 +5,38 @@ flow-nodes:
     description: Gets the type of data
     category: general
     methods:
-      getType:
+      getType: # This same key is used in `actions.js`
         name: Get Type
         description: Gets the type of data
         parameters:
           data:
+            name: Data
             description: The data that you want to find the type of.
             required: true
-        outputs:
-          # success output(s) should be first
-          next:
-            name: Next
-            context: $.type
             schema:
               type: string
-              enum:
-                - array
-                - bigint
-                - boolean
-                - function
-                - "null"
-                - number
-                - object
-                - string
-                - symbol
-                - undefined
+        # "Next" (next) output
+        returns:
+          name: Next
+          description: The operation was successful.
```

Comment:
This should be a more useful description related to the actual output in relation to this particular flow-node rather than something generic

Reply:
Is this description correct here?
<yaml>
returns:
          name: Next
          description: The typeof data is getting printed.``

Reply:
Description corrected.